source/manage/cloud-data-export.rst

```diff
@@ -1,8 +1,138 @@
-Cloud Data Export
-=========================
+Mattermost Workspace Migration
+==============================
 
-If, after completing your 14-day free Cloud trial, you've decided to self-host your own Mattermost deployment, you may want to migrate your Cloud data to your self-hosted instance.
+This document outlines the process for migrating from Mattermost Cloud to a Mattermost self-hosted instance. In the future, a process for migrating from Mattermost self-hosted to Mattermost Cloud will also be documented and provided here.
 
-We plan to publish documentation on how to perform a Cloud data export for channels, messages, users, and other bulk export data self-serve, with guidelines about running commands to export this data.
+Migrating between two installations follows the same process that's documented below, regardless as to whether the source or destination of the migration is in the Cloud or self-hosted. **These steps will work for any Mattermost instance**, as long as the instances are on Mattermost version 5.33 and up.
 
-In the meantime, if you're interested in migrating your Cloud data to self-hosted, please contact our support team via https://customers.mattermost.com/cloud/contact-us, or through the `Mattermost Help Center <https://support.mattermost.com/>`_. Once you've submitted your data export request, our team will `guide you through the process <https://handbook.mattermost.com/operations/research-and-development/engineering/cloud-data-export-process>`_. Typically the data export file will be provided within two weeks.
+Migrating from Cloud to self-hosted
+-----------------------------------
+
+If, after completing your 14-day free Cloud trial, you've decided to self-host your own Mattermost deployment, you can migrate your Cloud data to your self-hosted instance.
+
+How does the process work?
+--------------------------
+
+Before you export and migrate your data, you must `install Mattermost <https://docs.mattermost.com/guides/deployment.html#install-guides>`_ on the server you’ll be using to run Mattermost. The migration is done using the mmctl CLI tool, which is a remote CLI tool for Mattermost that's installed locally and uses the Mattermost API. From Mattermost version 6.0 onwards, mmctl is pre-installed. For versions prior to 6.0, first download the newest release from `the releases page <https://github.com/mattermost/mmctl/releases/tag/v6.1.0>`_, `install mmctl <https://docs.mattermost.com/manage/mmctl-command-line-tool.html#install-mmctl>`_, and place the binary on your ``PATH``.
```

Comment:
```suggestion
Before you export and migrate your data, you must `install Mattermost <https://docs.mattermost.com/guides/deployment.html#install-guides>`_ on the server you’ll be using to run Mattermost. The migration is done using the mmctl CLI tool, which is a remote CLI tool for Mattermost that's installed locally and uses the Mattermost API. From Mattermost v6.0 onwards, mmctl is pre-installed. For versions prior to 6.0, first download the newest release from `the releases page <https://github.com/mattermost/mmctl/releases/tag/v6.1.0>`_, `install mmctl <https://docs.mattermost.com/manage/mmctl-command-line-tool.html#install-mmctl>`_, and place the binary on your ``PATH``.
```

Reply:
I've created [a ticket on our v6.2 board](https://community-daily.mattermost.com/boards/workspace/g4k9s1f953n4bytz5f9sz9orer/bm1j7wcrrspoymkjermbydid97h/vcwyy7xnhh4qits4thrb3pte53e/cc3j1mtw7ytyhuckm5tceigwdqh) to update the mmctl docs to clarify that mmctl comes pre-installed as of v6.0 and later.